packages/grafana-ui/src/components/TimePicker/TimeRangePicker/TimeRangeList.tsx

```diff
@@ -29,10 +28,9 @@ const getOptionsStyles = stylesFactory(() => {
 interface Props {
   title?: string;
   options: TimeOption[];
-  value?: TimeRange;
-  onSelect: (option: TimeRange) => void;
+  value?: TimeOption;
+  onChange: (option: TimeOption) => void;
```

Comment:
We're changing an external interface here, right? Could it potentially break external plugins?

Reply:
No it is an internal component so it should be fine.